src/libraries/restore/runtime/runtime.depproj

```diff
@@ -90,6 +89,7 @@
       <BinPlaceItem Include="@(MonoIncludeFiles)">
         <DestinationSubDirectory>include/%(RecursiveDir)</DestinationSubDirectory>
       </BinPlaceItem>
+      <BinPlaceItem Remove="@(BinPlaceItem)" Condition="'$(RuntimeFlavor)' != 'Mono' and '%(BinPlaceItem.Filename)' == 'System.Private.CoreLib'" />
```

Comment:
Should we condition the mono binplaceitems above on RuntimeFlavor == Mono?

Reply:
We don't need to, as they would simply not be includded if you don't build for mono